qdrant-landing/content/documentation/integrations/cheshire-cat.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Cheshire Cat
+weight: 600
+---
+
+# Cheshire Cat
+
+[Cheshire Cat](https://cheshirecat.ai/) is an open-source framework that allows you to develop intelligent agents on top of many Large Language Models (LLM). You can develop your custom AI architecture to assist you in a wide range of tasks.
+
+<br>
+
+![Cheshire cat](https://cheshirecat.ai/wp-content/uploads/2023/11/cat.jpg)
+
+<br>
+
+## Cheshire Cat and Qdrant
+Cheshire Cat uses Qdrant as the default [Vector Memory](https://cheshire-cat-ai.github.io/docs/conceptual/memory/vector_memory/) for ingesting and retrieving documents.
+
+It uses:
+* memory based for testing
+* file based by default
+* container by settings two environment variables
+
+```
+# Decide host and port for your Cat. Default will be localhost:1865
+CORE_HOST=localhost
+CORE_PORT=1865
+
+# Qdrant server
+# QDRANT_HOST=localhost
+# QDRANT_PORT=6333
+```
+
+Cheshire Cat takes great advantage of the following features of Qdrant:
+* [Collection Aliases](https://qdrant.tech/documentation/concepts/collections/#collection-aliases) to manage the change from one embedder to another.
+* [Quantization](https://qdrant.tech/documentation/guides/quantization/) to obtain a good balance between speed, memory usage and quality of the results.
+* [Snapshots](https://qdrant.tech/documentation/concepts/snapshots/) to not miss any information.
+* [Community](https://discord.com/invite/tdtYvXjC4h) and [documentation](https://qdrant.tech/documentation/)
```

Comment:
Please use local links for local resources, such as:

```diff
-* [Collection Aliases](https://qdrant.tech/documentation/concepts/collections/#collection-aliases) to manage the change from one embedder to another.
+* [Collection Aliases](../../concepts/collections/#collection-aliases) to manage the change from one embedder to another.

Reply:
Thanks, done.